src/test/ui/half-open-range-patterns/feature-gate-half-open-range-patterns.rs

```diff
@@ -0,0 +1,21 @@
+#![feature(exclusive_range_pattern)]
+
+fn main() {}
+
+#[cfg(FALSE)]
```

Comment:
why this cfg? To test that these things are checked in the parser?

Reply:
Yep; standard trick to check that pre-expansion gating happens.